app/src/lisk_base32.c

```diff
@@ -34,3 +34,26 @@ uint8_t lisk_base32_encode(const uint8_t *input, const uint8_t inputLen,
 
     return inputLen;
 }
+
+unsigned char lisk_encode_varint(unsigned long int value, unsigned char *dest) {
+    uint8_t tmp;
+    if (value <= 0xfc) {
+        memmove(dest, &value, 1);
+        return 1;
+    } else if (value <= 0xffff) {
+        tmp = 0xfd;
+        memmove(dest, &tmp, 1);
+        memmove(dest + 1, &value, 2);
+        return 3;
+    } else if (value <= 0xffffffff) {
```

Comment:
this condition will be always true. Maybe we can just leave the `else`. Besides, `0xffff` can be replaces by `UINT16_MAX`